metadata-ingestion/src/datahub/ingestion/source/csv_enricher.py

```diff
@@ -546,7 +546,9 @@ def maybe_extract_glossary_terms(
         term_urns: List[str] = terms_array_string.split(self.config.array_delimiter)
 
         term_associations: List[GlossaryTermAssociationClass] = [
-            GlossaryTermAssociationClass(term) for term in term_urns
+            GlossaryTermAssociationClass(term)
+            for term in term_urns
+            if term.startswith("urn:li:")
```

Comment:
Is this just due to an ill-formatted file? Can we instead append `url:li` if it doesn't start with that, like we do elsewhere in `make_data_platform_urn` for example

Reply:
No that is not possible. People are using this for terms while their actual URNs are UUIDs.

Reply:
Feel free to stamp this one then. I don't understand what's going on here